sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -861,20 +861,17 @@ def collect_metrics(self):
 
                         instance_results[cls] = rows, cols
 
-                # Using the cached data, extract and report individual metrics
                 for metric in self.instance_metrics:
-                    if type(metric) is metrics.SqlIncrFractionMetric:
-                        # special case, since it uses the same results as SqlFractionMetric
-                        key = 'SqlFractionMetric'
-                    else:
-                        key = metric.__class__.__name__
-
```

Comment:
> # special case, since it uses the same results as SqlFractionMetric

this comment is incorrect, and this logic resulted in a bug, which resulted in us never actually reporting metrics that fell into the `SqlIncrFractionMetric` category